packages/ag-charts-community/src/chart/series/cartesian/barSeries.ts

```diff
@@ -446,6 +447,7 @@ export class BarSeries extends AbstractBarSeries<
                         : undefined,
                 missing: yValue == null,
                 focusable: !phantom,
+                style: this.getItemStyle({ xValue, yValue, datum, datumIndex, phantom }, false),
```

Comment:
I think we have a mix of stuff in this style property and outside. Like `opacity`, `clipBBox`, `crisp` etc. Did you have any thoughts on how you determine what goes in a style vs. not?